src/Core/src/Core/IIndicatorView.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui
+{
+	public interface IIndicatorView : IView
+	{
+		int Count { get; }
+
+		int Position { get; set; }
+
+		double IndicatorSize { get; }
+
+		int MaximumVisible { get; }
+
+		bool HideSingle { get; }
+
+		Paint? IndicatorColor { get; }
+
+		Paint? SelectedIndicatorColor { get; }
+
+		IShapeView IndicatorsShape { get; }
```

Comment:
I wonder about the type choices here. 

If this needs to be an `IShapeView` then it feels like we'd just make this an IView and then in the code you'd check if it was an IShapeView or an ILayout and do the correct thing with it?

Should this be an `IShape` instead? 

Or should we have a type for the Indicator like we we have with `Brushes`

`class Indicator`
`class IndicatorShape`
`class TemplatedIndicatorShape`

and then the code would know how to build each? 

Most of the code seems like it could use the type of this value to branch instead of needing to check the interface?

Reply:
it desn't need to be a IShapeVIew, i just need to know if it's a circle or square but didn't want to add to core IndicatorShape enum .

Reply:
Well it's not a IShape, because theres no Square: IShape, we just  add it to know how to draw circles or squares in theNative, but i think i will just move IndicatorShape to Core .. and it would be easier.

Reply:
Did some refactor it's now a IShape